lib/synced/strategies/full.rb

```diff
@@ -192,6 +192,15 @@ def search_params
         end]
       end
 
+      def scope_params
+        case @scope
+        when Rental
```

Comment:
we should have this `Synced::ActsAsScope`? required to be used as scope and with
```
def synced_scope_params
  raise NotImplementedError # or {}
end
```
?